extension/src/plots/model/index.ts

```diff
@@ -112,8 +112,6 @@ export class PlotsModel extends ModelWithPersistence {
 
```

Comment:
Probably an issue on my end, but when I run an experiment, the plots break with this error: `plots diff dolce-hugs techy-hugs 53aac68fc16182b38c509da7554c22f822fc8d63 miffy-runs -o .dvc/tmp/plots --split --json failed with ERROR: unknown Git revision 'miffy-runs'`. 

https://user-images.githubusercontent.com/43496356/232789285-4a7d06cd-793c-4325-b92c-05d88d6cedb4.mov

I can't see plots until the experiment finishes running.

Reply:
I thought I had tested this but obviously not. Seems like a bug in DVC. I'll confirm whether or not is expected and if they can provide a workaround. If not we'll be going back to the old behaviour.